### DISCUSSION TITLE
got rejected it says:- Please upload proof of your current school affiliation

### DISCUSSION THREAD

**Discussion by thisisforlee** [community/community#54020] [GitHub Education]
### Hi everyone,

"Please upload proof of your current school affiliation" who I uploaded my original collage id ?

**Comment by Siloshah** [comment]
By mistake my wallet address samit rong plz chang it

**Comment by samus-aran** [comment]
Hi @thisisforlee 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Reply by harshwardanrajpurohit** [reply]
i have uplloaded  Bonofide But its still failed

**Comment by PRAFULPAWAR8888** [comment]
same problem..., Can it fix bro
